client/src/components/SVGIcon/SVGIcon.scss

```diff
@@ -50,6 +50,69 @@
   }
 }
 
+.application-incident-metadata-icon {
+  @extend .svg-icon;
+  mask: url('/src/assets/application-incident-metadata-icon.svg') no-repeat center;
+
+  .svg-icon {
+    background-color: $green;
+  }
```

Comment:
Maybe we should make green by default and override it only if we need different color?